scripts/update_and_rebuild_libmesh.sh

```diff
@@ -86,8 +92,7 @@ if [ -z "$go_fast" ]; then
                --enable-unique-ptr \
                --enable-openmp \
                --disable-maintainer-mode \
-               --enable-petsc-required \
```

Comment:
Nope.  You deleted the `--enable-petsc-required` line I just added, possibly during conflict resolution?  Please put that back.